Utility/Utility.Test/UtilityTimeTest.cpp

```diff
@@ -0,0 +1,29 @@
+#include <catch2/catch.hpp>
+#include <ctime>
+
+#include "Utility/UtilityTime.hpp"
+
+using Catch::Matchers::Contains;
+
+TEST_CASE("consoleLogTimestamp test")
+{
+    std::ostringstream output_time;
+    auto               cout_buff = std::cout.rdbuf();  // save pointer to std::cout buffer
+    std::cout.rdbuf(output_time.rdbuf());              // substitute internal std::cout buffer with buffer of 'local' object
+
+    //getting the time
+    std::chrono::system_clock::time_point currentTime = std::chrono::system_clock::now();
```

Comment:
Use _using_ to decrease length of code.
_using std::chrono::system_clock::time_point;_